src/main/java/io/dropwizard/flyway/FlywayFactory.java

```diff
@@ -21,6 +21,8 @@
     @NotEmpty
     private String encoding = StandardCharsets.UTF_8.name();
     @JsonProperty
+    private String defaultSchema = null;
```

Comment:
Could we mark this field and the corresponding getter and setter as `@Nullable`?

See `target` for an example.

Reply:
@joschi yep, I've updated it to include `@Nullable` in https://github.com/dropwizard/dropwizard-flyway/pull/81/commits/528c818c6407a42985931a784bb0bd4074103fa2.  I also moved the `flyway.defaultSchema(defaultSchema);` to a separate `if` block in `createConfiguration()` to match the pattern of the other nullable fields.  Thanks!